### PR TITLE
Adds Jaywalkers

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Adds Jaywalking as a prisoner crime
Also moved Mass Murder so it's alphabetically ordered.

## Why It's Good For The Game

I was pretty surprised such a serious crime was somehow overlooked.

But more seriously, I feel it fits the tone of this game to have something a little silly there. Hopefully the warden and sec won't be too hard on people put away for this crime.

## Changelog

:cl:
add: Prisoners can now be jaywalkers
qol: reordered a crime so it appears on the choice list alphabetically.
/:cl:

### CODE REVIEW

**Comment by TheVekter** [comment]
You should include something about walking through tram crossings when they're not green on maps with trams.

**Comment by YehnBeep** [comment]
> You should include something about walking through tram crossings when they're not green on maps with trams.

I don't know enough to make it map-specific but since the crimes would presumably have been committed previously and over time anyways - I added a mention.

File: code/modules/client/preferences/prisoner_crime.dm [hunk 1]
```diff
@@ -98,12 +98,22 @@ GLOBAL_LIST_INIT(prisoner_crimes, init_prisoner_crimes())
 	name = "Identity Theft of High-Ranking Figure"
 	desc = "Impersonated a high-ranking figure."
 	tattoos = 0 //well, obviously can't impersonate people with tats. if they want to go back to doing that
+	
+/datum/prisoner_crime/jaywalker
+	name = "Jaywalker"
+	desc = "Jaywalked across non-green tram crossings, shuttle docking zones, and through space."
```
Comment (BlueMemesauce) [inline code/modules/client/preferences/prisoner_crime.dm:104]:
```suggestion
	desc = "Jaywalked across non-green tram crossings, shuttle docking zones, or through space."
```
you could also put and/or

Comment (YehnBeep) [reply code/modules/client/preferences/prisoner_crime.dm]: 👍

**Pull request merged**